identifiers/country-us/state-ar-local_gov.csv

```diff
@@ -324,9 +541,25 @@ ocd-division/country:us/state:ar/county:perry/council_district:6,Perry County Di
 ocd-division/country:us/state:ar/county:perry/council_district:7,Perry County District 7
 ocd-division/country:us/state:ar/county:perry/council_district:8,Perry County District 8
 ocd-division/country:us/state:ar/county:perry/council_district:9,Perry County District 9
+ocd-division/country:us/state:ar/county:phillips/council_district:1,Phillips County District 1
+ocd-division/country:us/state:ar/county:phillips/council_district:2,Phillips County District 2
+ocd-division/country:us/state:ar/county:phillips/council_district:3,Phillips County District 3
+ocd-division/country:us/state:ar/county:phillips/council_district:4,Phillips County District 4
+ocd-division/country:us/state:ar/county:phillips/council_district:5,Phillips County District 5
+ocd-division/country:us/state:ar/county:phillips/council_district:6,Phillips County District 6
+ocd-division/country:us/state:ar/county:phillips/council_district:7,Phillips County District 7
+ocd-division/country:us/state:ar/county:phillips/council_district:8,Phillips County District 8
+ocd-division/country:us/state:ar/county:phillips/council_district:9,Phillips County District 9
```

Comment:
It looks like there are 11 districts for Phillips County: http://phillips.ark.org/directory